app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,14 @@
+do ->
+  $(document).on 'turbolinks:load', ->
+    init_selectized_roles_dropdown = ->
+      $('.select-role').selectize maxItems: 3
+    init_form_auto_submit = ->
+      $('select.select-role').change ->
+        $(this).closest("form").submit()
+    init_delete_user_button = ->
```

Comment:
That's good. Now you can move function definitions outside of `$(document).on 'turbolinks:load'` and leave only their usage.